store/use-reservation-store.ts

```
@@ -0,0 +1,118 @@
+        reservation.name.toLowerCase().includes(lowerSearchTerm) ||
+        reservation.email.toLowerCase().includes(lowerSearchTerm) ||
+        reservation.locator.toLowerCase().includes(lowerSearchTerm)
+    );
+  },
+  
+  getTotalCommission: () => {
+    const { reservations } = get();
+    const total = reservations
+      .filter(res => res.extras.includes('reservados'))
+      .reduce((sum, res) => {
+        const itemCount = parseInt(res.extras.split(' ')[0]) || 0;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)